README.md

```diff
@@ -24,6 +24,49 @@ Deploying supabase via Kubernetes allows for a cloud agnostic deployment as long
 
 See [Official Documentation](https://supabase.com/docs)
 
+## Running locally
+
+To run this app locally, follow these steps:
+
+1. Install [Android Studio](https://developer.android.com/studio)
+2. Run Android Studio, which will help you install the Android toolchain. Be sure to include all required components
+   1. Android SDK Platform
+   1. Android SDK Command-line tools
+   1. Android SDK Build-Tools
+   1. Android SDK Platform-Tools
+   1. Android Emulator
+3. Install Pixi: https://github.com/prefix-dev/pixi?tab=readme-ov-file#installation
+4. In the package's directory, run
+```
+pixi run install-tools
+```
```

Comment:
```suggestion
4. In the package's directory, run
    
    ```console
    pixi run install-tools
    ```
```

Could you modify the formatting a little bit for these.

Reply:
done :)